content/src/service/access/AccessCheckerForWearables.ts

```diff
@@ -115,12 +136,17 @@ export class AccessCheckerForWearables {
       itemId,
       block
     )
+
     const ethAddressLowercase = ethAddress.toLowerCase()
-    return (
+    const addressHasAccess =
       (permissions.collectionCreator && permissions.collectionCreator === ethAddressLowercase) ||
       (permissions.collectionManagers && permissions.collectionManagers.includes(ethAddressLowercase)) ||
       (permissions.itemManagers && permissions.itemManagers.includes(ethAddressLowercase))
-    )
+
+    // Deployments to the content server are made after the collection is completed, so that the committee can then approve it
+    const isCollectionValid = !permissions.isApproved && permissions.isCompleted
```

Comment:
is it okay to negate `!permissions.isApproved`? If so, can you make it more clear in the comment, please?

Reply:
Just made the comment clearer